app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/EditActivity.java

```diff
@@ -85,10 +87,14 @@ public static Intent buildIntent(Context context, ImageDetails imageDetails) {
     @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
     final EditTagListener editTagListener = new EditTagListener() {
         @Override
-        public void openEditDialog(Tag tag) {
+        public void openTagEditionView(Tag tag) {
             if (tag.getKey().equals(ExifInterface.TAG_GPS_LATITUDE)) {
```

Comment:
better approach is comparing constant value with sth - we are securing against NUllPointerException